wsgi.py

```diff
@@ -10,16 +10,52 @@
 import book_webscrape
 import movie_webscraper
 import os
+import base64
 
 app = Flask(__name__)
 
 @app.route('/weather/api/v1/<lat>&<long>', methods=['GET'])
 
 def weather(lat, long):
+
+    spotify_id = os.getenv('SPOTIFY_CLIENT_ID')
+    spotify_secret = os.getenv('SPOTIFY_CLIENT_SECRET')
+
+    userpass = spotify_id + ':' + spotify_secret
+    encoded_u = base64.b64encode(userpass.encode()).decode()
+
+    spotify_url = "https://accounts.spotify.com/api/token"
+
+    payload = 'grant_type=client_credentials'
+    headers = {
+      'Authorization': "Basic %s" % encoded_u,
+      'Content-Type': 'application/x-www-form-urlencoded'
+    }
+
+    spotify_response = requests.request("POST", spotify_url, headers=headers, data = payload)
+    spotify_token = (spotify_response.text.encode('utf8'))
+
+    spotify_token_json = json.loads(spotify_token)
+    spotify_token_json['access_token']
+    spotify_token = spotify_token_json['access_token']
+
```

Comment:
Great work on getting the Spotify API call. We have an opportunity to add test coverage for this method in a separate file!